Typos and doc tweaks (#1181)
microsoft/vs-streamjsonrpc: doc/sendrequest.md

which StreamJsonRpc supports. Notifications cannot be canceled.

A client can cancel a request by canceling a `CancellationToken` supplied when originally invoking the request.

> Be sure to call the `JsonRpc.Invoke*` method and an overload that explicitly accepts `CancellationToken`

Author: How about:

```suggestion
Be sure to call an overload of a `JsonRpc.Invoke*` method that explicitly accepts `CancellationToken`
```
